cmd/adapter-rest/startcmd/start.go

```diff
@@ -60,14 +60,8 @@ const (
 
 // API endpoints.
 const (
-	healthCheckEndpoint                = "/healthcheck"
-	hydraLoginEndpoint                 = "/login"
-	hydraConsentEndpoint               = "/consent"
-	oidcCallbackEndpoint               = "/callback"
-	createPresentationRequestEndpoint  = "/presentations/create"
-	handlePresentationResponseEndpoint = "/presentations/handleResponse"
-	userInfoEndpoint                   = "/userinfo"
-	uiEndpoint                         = "/ui"
+	healthCheckEndpoint = "/healthcheck"
+	uiEndpoint          = "/ui"
```

Comment:
why isn't this one under `pkg`?

Reply:
I was thinking of moving healthCheckEndpoint

Reply:
I moved healthCheckEndpoint